website/docs/r/alert_muting_rule.html.markdown

```diff
@@ -45,6 +45,7 @@ The following arguments are supported:
   * `enabled` - (Required) Whether the MutingRule is enabled.
   * `name` - The name of the MutingRule.
   * `description` - The description of the MutingRule.
+  * `schedule` - (Optional) Specify a schedule for enabling the MutingRule. See [Schedule](#schedule) below for details
```

Comment:
Since this feature seems to be gaining interesting, can we add an example showing the use of `schedule` in a section titled "Additional Example" at the bottom of this doc file? Here's an [example from NRQL alert conditions](https://github.com/newrelic/terraform-provider-newrelic/blob/master/website/docs/r/nrql_alert_condition.html.markdown#additional-examples).